### PR TITLE
metrics: add grafana and alertmanager scripts

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### What problem does this PR solve? 
Maintain the grafana and alert manager scripts in PD repo.

### What is changed and how it works?
Add grafana and alert manager scripts.

### Check List 

Tests 

 - No code

### CODE REVIEW

File: metrics/grafana/pdn.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
+         {
```
Comment (nolouch) [inline metrics/grafana/pdn.json]: We can remove `pdn.json`, it's not used anywhere.

Comment (nolouch) [inline metrics/grafana/pdn.json]: We can remove `pdn.json`, it's not used anywhere.

**Comment by rleungx** [comment]
Why do we move these files into PD?

**Comment by nolouch** [comment]
Will ansible fetch the rules from PD repo?

**Comment by qiffang** [comment]
> Will ansible fetch the rules from PD repo?

We have a tool to fetch monitoring configurations from each repo

**Comment by qiffang** [comment]
> To ensure synchronization, we will delete pd.json in tidb-ansible?

Yes

**Comment by lhy1024** [comment]
> > To ensure synchronization, we will delete pd.json in tidb-ansible?
> 
> Yes

Good. Then we need to remember merging [it](https://github.com/pingcap/tidb-ansible/pull/1097)

**Pull request merged**